themes/gatsby-theme-blog/src/pages/404.js

```diff
@@ -12,8 +13,10 @@ class NotFoundPage extends React.Component {
     return (
       <Layout location={this.props.location} title={siteTitle}>
         <SEO title="404: Not Found" />
-        <h1>Not Found</h1>
-        <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
+        <Styled.h1>Not Found</Styled.h1>
+        <Styled.p>
+          You just hit a route that doesn&#39;t exist... the sadness.
```

Comment:
This is something I keep forgetting to do, which is to update the 404 copy to be more helpful. Happy to skip this for another PR, but figured if you don’t mind we can just tack it on here.
```suggestion
          The page you’re trying to visit has moved or doesn’t exist. Please check the URL or use the site navigation to find the content you’re looking for.
```